packages/EasyTest/src/Trait/Messenger/MessengerAssertionsTrait.php

```diff
@@ -0,0 +1,211 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyTest\Trait\Messenger;
```

Comment:
Please fix to `Traits` to be consistent.

Reply:
Fixed